### PR TITLE
BreakClearLevel fix

### PR DESCRIPTION
**Pull request by mf022**
BreakClearing only in the case of MFHelper
additional fix for https://github.com/blizzhackers/kolbot/pull/31

### CODE REVIEW

File: d2bs/kolbot/libs/bots/MFHelper.js [hunk 1]
```diff
@@ -236,5 +237,6 @@ MainLoop:
 		delay(100);
 	}
 
+	Config.MFHelper.BreakClearLevel = false; // revert to default value
```
Comment (jaenster) [inline d2bs/kolbot/libs/bots/MFHelper.js]: You cant assume this is the value that was set

File: d2bs/kolbot/libs/common/Config.js [hunk 2]
```diff
@@ -436,7 +436,7 @@ var Config = {
 		RecheckSeals: false
 	},
 	MFHelper: {
-		BreakClearLevel: true
+		BreakClearLevel: false
```
Comment (jaenster) [inline d2bs/kolbot/libs/common/Config.js]: this is a breaking change, why the switch?

Comment (mf022) [reply d2bs/kolbot/libs/common/Config.js]: https://d2bot.discourse.group/t/break-the-clearing/614

Comment (mf022) [reply d2bs/kolbot/libs/common/Config.js]: to have the BreakClearLevel only in the case of executing MFHelper, which will stop the clearing in area if leader change that to other clearing area.

**Comment by 5noop** [comment]
Consider the following change in Attack.js instead:
```
if (Loader.scriptName() === "MFHelper" && Config.MFHelper && Config.MFHelper.BreakClearLevel && Config.Leader !== "") {```
https://github.com/blizzhackers/kolbot/blob/3abdafae46100d91f80aa3bcf3073be8a762998b/d2bs/kolbot/libs/common/Attack.js#L901

File: d2bs/kolbot/libs/common/Config.js (hunk 2 above)
Comment (noah-) [reply d2bs/kolbot/libs/common/Config.js]: why was BreakClearLevel even added? this should be handled differently altogether, not be a config option
https://github.com/blizzhackers/kolbot/pull/31/files

File: d2bs/kolbot/libs/bots/MFHelper.js (hunk 1 above)
Comment (mf022) [reply d2bs/kolbot/libs/bots/MFHelper.js]: even @kolton used the overwrite of Config values - https://github.com/blizzhackers/kolbot/blob/master/d2bs/kolbot/libs/bots/Follower.js#L548-L551

File: d2bs/kolbot/libs/common/Config.js (hunk 2 above)
Comment (mf022) [reply d2bs/kolbot/libs/common/Config.js]: anyone with a better idea could implement that in a new PR which will fix all issues related to the MFHelper

**Comment by Fa-b** [comment]
Agree with @5noop 's suggestion for now. `&& Config.MFHelper` could be removed as a guard also.
No need for hardcoded config overwrites in MFHelper.js. The option should still be default false imo.

**Comment by Fa-b** [comment]
see #126. This here seems inactive

**Pull request closed without merging**